dashboard/app/views/levels/editors/_gamelab.html.haml

```diff
@@ -6,19 +6,11 @@
 
 = render partial: 'levels/editors/fields/debugger', locals: {f: f}
 
-= render partial: 'levels/editors/fields/control_buttons', locals: {f: f}
-
-= render partial: 'levels/editors/fields/submittable', locals: {f: f}
-
-= render partial: 'levels/editors/fields/key_value_data', locals: {f: f}
-
-.field.aligned-input-group
-  = boolean_check_box f, :hide_view_data_button
-  = f.label 'Hide View Data Button'
-
-= render partial: 'levels/editors/fields/validation_code', locals: {f: f}
-
-= render partial: 'levels/editors/fields/animation', locals: {f: f}
+%legend.control-legend.collapsed{data: {toggle: "collapse", target: "#display-area"}}
+  Display Area
+#display-area.in.collapse
```

Comment:
I think the star labs team uses the term "Visualization area" to refer to the left pane where you see the app running. if that's what you're referring to here, please change these to say "Visualization area" instead of "Display area", unless you think that'd be more confusing for curriculum writers. 

either way, a short description below this header could be helpful to clarify what area you're talking about.

Reply:
Based on your feedback I ungrouped these fields.